frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetrics.tsx

```diff
@@ -22,26 +21,41 @@ import {
     withDefault,
 } from 'use-query-params';
 import { aggregateFeatureMetrics } from './aggregateFeatureMetrics';
-import { useExtendedFeatureMetrics } from './useExtendedFeatureMetrics';
 
 export const FeatureMetrics = () => {
     const projectId = useRequiredPathParam('projectId');
     const featureId = useRequiredPathParam('featureId');
     const environments = useFeatureMetricsEnvironments(projectId, featureId);
-    const applications = useFeatureMetricsApplications(featureId);
```

Comment:
applications we get depend on time span now so we can't select the longest anymore. daily timespans are aggregated with up to 24h delay so we may not have some apps in the daily timespans